jekyll/_cci2/pipelines.adoc

```diff
@@ -0,0 +1,198 @@
+
+=== Configuration source
+
+Configuration sources define the units of work that can be run within your projects. Each configuration source defines the location of a pipeline configuration file, and can be set up to be used when _triggering_ a pipeline. Configuration sources can be associated with one or more triggers.
+
+A project can have one or more configuration sources. The default location for a CircleCI configuration file is `.circleci/config.yml`. If you authenticated through the GitHub App, you can set the location to _any_ path to a `*.yml` file.
+
+When you create a project in CircleCI and connect it to a repository, a configuration source is set up for you automatically, and connected to a configuration file. The connected configuration file will be either a file you commit  yourself or one generated by CircleCI based on the code in your project.
+
+image::{{site.baseurl}}/assets/img/docs/project-settings-configuration.png[Configuration source]
+
+==== Add a configuration source
+
+TIP: If you created a project xref:create-project#[in the CircleCI web app] you will find a configuration source is already set up.
+
+image::{{site.baseurl}}/assets/img/docs/add-configuration-source.png[Add a configuration source]
+
+To add a configuration source, follow these steps:
+
+. In the CircleCI web app, click on **Projects** in the sidebar
+. Click on the ellipsis next to your project (icon:ellipsis-h[]) and then click **Project Settings**
+. Select **Configuration** in the sidebar
+. Click btn:[Add Configuration Source]
+. Complete the form fields and options:
+** Give your configuration source a descriptive name, for example `Run tests and deploy`
+** Under Configuration Source select the platform that matches where your project is set up (for example, GitHub App)
++
+NOTE: For GitLab Self-managed, you can input any instance that you have previously set up in CircleCI. If you wish to set a different feature branch or repository from a self-managed instance as a new configuration source, you will first need to add a new connection via your xref:gitlab-integration#organization-settings-integrations[menu:Organization Settings[Integrations]]. In either case, you will also need to enter your personal access token again to authorize this connection.
+** Authorize your connection if this is not already showing with a icon:check-circle[]
+** Select your repository
++
+CAUTION: **Select the same repository for your configuration source and trigger** (described in the following section). There is currently a bug that means that CircleCI will always look for the configuration source in the same repository from which the triggering event was emitted.
+** (Optional) If you will be assigning an inbound webhook trigger to this configuration source, select a fallback branch to use in case the triggering event does not have a triggering branch.
```

Comment:
It's not optional :( 
Suggestion:
``` 
Select a fallback branch to use in case the triggering event does not have a triggering branch. This branch will be used if you trigger this pipeline via inbound webhook
```